packages/terraform-modules/testnet/main.tf

```diff
@@ -0,0 +1,92 @@
+provider "google" {
+  credentials = file("~/.gcloud/service-accounts/celo-testnet-69a4cd9692dd.json")
+  project     = "celo-testnet"
+  region      = "us-west1"
+  zone        = "us-west1-a"
+}
+
+# For managing terraform state remotely
+terraform {
+  backend "gcs" {
+    bucket = "celo_tf_state"
+  }
+}
+
+data "terraform_remote_state" "state" {
+  backend = "gcs"
+  config = {
+    bucket = "celo_tf_state"
+    prefix = "${var.celo_env}/state"
+  }
+}
+
+resource "google_compute_network" "network" {
+  name = "${var.celo_env}-network"
+}
+
+resource "google_compute_firewall" "ssh_firewall" {
+  name = "${var.celo_env}-ssh-firewall"
+  network = google_compute_network.network.name
+
+  allow {
+    protocol = "tcp"
+    ports = ["22"]
+  }
+}
+
+resource "google_compute_firewall" "geth_firewall" {
```

Comment:
Do we have an equivalent of port-forwarding here? I.e. what does it look like for someone to want to deploy smart contracts or faucet or send invites here?

Reply:
Just kidding, saw that you already planned for that in your PR description!